agent/exec/container/executor.go

```diff
@@ -41,12 +41,29 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 		}
 	}
 
+	// add v1 plugins to 'plugins'
 	addPlugins("Volume", info.Plugins.Volume)
 	// Add builtin driver "overlay" (the only builtin multi-host driver) to
 	// the plugin list by default.
 	addPlugins("Network", append([]string{"overlay"}, info.Plugins.Network...))
 	addPlugins("Authorization", info.Plugins.Authorization)
 
+	// retrieve v2 plugins
+	v2plugins, err := e.client.PluginList(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// add v2 plugins to 'plugins'
+	for _, plgn := range v2plugins {
+		for _, typ := range plgn.Config.Interface.Types {
+			plugins[api.PluginDescription{
+				Type: typ.String(),
```

Comment:
`typ.String()` will return something of the format `docker/VolumeDriver:1.0`. However, test code in scheduler_test.go: L1404 still sets the type to be `Volume`. Where are the changes in swarmkit to accomodate this new format?